java-tools/OsmAndServer/src/main/java/net/osmand/server/api/test/dto/DatasetRecord.java

```diff
@@ -0,0 +1,25 @@
+package net.osmand.server.api.test.dto;
```

Comment:
package name is unclear net.osmand.server.api.test -> net.osmand.server.api.searchtest

Reply:
By codestyle it's preferred to have classes at least 50-100 LoC, so this mini classes could be moved as internal classes to repositories also minimizing number of packages